NBitcoin/Protocol/Node.cs

```diff
@@ -311,6 +318,11 @@ private void Cleanup(Exception unhandledException)
 				}
 			}
 
+			internal void CleanUp()

```

Comment:
Whats the difference between endlisten and cleanup?

Reply:
I renamed Cleanup to EndListen as the *current* cleanup doesn't *reaaaally* cleanup per se, its more like finalize method.

But I needed to create a Cleanup for NodeConnection to dispose the ManualResetEvent and CancellationTokenSource objects.